### PR TITLE
back to using for nonextended functions

### PR DESCRIPTION
**Pull request by sbromberger**

### CODE REVIEW

**Comment by sbromberger** [comment]
The whitespace changes are due to vscode autoformat.

File: benchmark/core.jl [hunk 1]
```diff
@@ -1,13 +1,13 @@
 function bench_iteredges(g::AbstractGraph)
```
Comment (sbromberger) [inline benchmark/core.jl]: whitespace changes only in this file.

**Comment by sbromberger** [comment]
cc @jpfairbanks @StefanKarpinski @juliohm @mbesancon @issamt @SohamTamba @somil55

**Comment by sbromberger** [comment]
Summary of changes:

As it is now, after #895, we were using `import Foo` for all packages. This has the effect of requiring fully-qualified names for both extending and using functions: `Foo.do_something(a, b, c)`, `Foo.do_other_thing()`.

It was suggested that a better way of doing this was to use `using Foo: do_something, do_otherthing` for methods that weren't being extended, and `import Foo: extend_this_method` for methods that were. This has the advantage of only bringing into the namespace those functions/methods that are required, and also allows us to avoid fully-qualifying the external functions wherever they're called.

So, in effect, this PR reverts and moves us to a previously-unmentioned option 4.

**Comment by sbromberger** [comment]
Folks, I'd appreciate a review on this at some point. Thank you.

File: src/traversals/greedy_color.jl [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 Store number of colors used and mapping from vertex to color
 """
-struct coloring{T<:Integer} <: Any
+struct coloring{T <: Integer} <: Any
```
[the remark this answers is not in the document]
Comment (sbromberger) [reply src/traversals/greedy_color.jl]: No. But this isn't a change to this PR, so we shouldn't address it here.

File: src/edit_distance.jl [hunk 3]
```diff
@@ -41,66 +41,66 @@ if involved costs are equivalent.
...
-  DataStructures.enqueue!(OPEN, [(1, 0)], delete_cost(1) + h([(1, 0)]))
+    OPEN = PriorityQueue{Vector{Tuple},Float64}()
+    for v in 1:nv(G₂)
+        enqueue!(OPEN, [(1, v)], subst_cost(1, v) + h([(1, v)]))
+    end
+    enqueue!(OPEN, [(1, 0)], delete_cost(1) + h([(1, 0)]))
 
-  while true
+    while true
     # minimum (partial) edit path
```
[the remark this answers is not in the document]
Comment (mbesancon) [reply src/edit_distance.jl]: (not with the while)

File: src/traversals/diffusion.jl [hunk 4]
```diff
@@ -17,12 +17,12 @@ from a vertex ``i`` to each of the `outneighbors` of ``i`` to
 ``\\frac{p}{outdegreee(g, i)}``.
 """
 function diffusion(g::AbstractGraph{T},
-                   p::Real,
-                   n::Integer;
-                   watch::AbstractVector=Vector{Int}(),
-                   initial_infections::AbstractVector=LightGraphs.sample(vertices(g), 1),
-                   normalize::Bool=false
-                   ) where T
+                    p::Real,
```
[the remark this answers is not in the document]
Comment (mbesancon) [reply src/traversals/diffusion.jl]: it seemed so when I cloned the PR

**Comment by sbromberger** [comment]
Folks, I'm going to go ahead and merge this tomorrow. Last call for reviews (thanks, @mbesancon, for your comments. I think you've found some things that should be fixed in a separate PR.)

**Pull request merged**